src/components/tempNews.js

```diff
@@ -40,7 +40,7 @@ class News extends React.Component {
         // eslint-disable-next-line
         <marquee className="space-x-2" scrollamount="10">
           {this.state.data.articles.map((item) => (
-            <p className="inline text-2xl" key={item.key}>{item.title} </p>
+            <p className="inline text-6xl font-semibold" key={item.key}>{item.title} </p>
```

Comment:
This'll be too big! Probably say a size above what I had set (two at a push) is where it should be.

Make the video bigger if you want to fill the space